components/ntp_background_images/browser/features.h

```diff
@@ -20,11 +21,11 @@ BASE_DECLARE_FEATURE(kBraveNTPBrandedWallpaper);
 
 // Show initial branded wallpaper after nth new tab page for fresh opens.
 inline constexpr base::FeatureParam<int> kInitialCountToBrandedWallpaper{
-    &kBraveNTPBrandedWallpaper, "initial_count_to_branded_wallpaper", 1};
+    &kBraveNTPBrandedWallpaper, "initial_count_to_branded_wallpaper", 2};
```

Comment:
Desktop/Android previously required the default to be nth tab minus 1. This was not the case for iOS and makes more sense, so for now I have subtracted 1 where desktop/Android uses this feature param. A followup PR for desktop/Android will remove the need to subtract 1.